app/src/main/java/fr/free/nrw/commons/profile/leaderboard/UserDetailAdapter.java

```diff
@@ -84,6 +93,28 @@ public void onBindViewHolder(@NonNull UserDetailAdapter.DataViewHolder holder, i
             holder.getContext().getResources().getString(R.string.count_prefix),
             leaderboardResponse.getCategoryCount()));
 
+        // When user tap on avatar shows the toast on how to change avatar
+        // fixing: https://github.com/commons-app/apps-android-commons/issues/47747
+        if (currentLoginUserName == null) {
+            // check current login username is not fetched then fetch it
```

Comment:
Please rephrase: `If the current login username has not been fetched yet, then fetch it.